src/js/components/form-elements/ModalWrapper.jsx

```diff
@@ -169,6 +178,7 @@ ModalWrapper.propTypes = {
 
 ModalWrapper.defaultProps = {
   btnOpenText: 'default.button.open.label',
+  btnOpenDefaultText: 'default.button.open.label',
```

Comment:
the default text should not be a code, but translated text ("Open" in this case)